installers/linux/deb/kvm2_deb_template/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: docker-machine-driver-kvm2
+Version: --VERSION--
+Section: base
+Priority: optional
+Architecture: amd64
+Depends: libvirt0 (>= 1.3.1)
+Recommends: minikube
+Maintainer: Aaron Prindle <aaprindle@gmail.com>
```

Comment:
This should probably be someone else. If you don't want your name, you may put in my own address: `t+minikube@stromberg.org`

I wonder if Debian accepts mailing lists as an alternative to a single person?

Reply:
Okay, will change the maintainer on both packages (minikube and docker-machine-driver-kvm2)

I think that it does, the `docker-ce` package has: `Maintainer: Docker <support@docker.com>`